edb/edgeql/tracer.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, name, *, source=None, target=None):
         self.target = target
         self.pointers = {}
 
-    def is_pointer(self):
+    def is_pointer(self) -> bool:
```

Comment:
Speaking of typing, could you please strict-type the rest of this file and related files in a follow up PR?

Reply:
OK, I'll try typing this file.